fastlane/Fastfile

```diff
@@ -711,6 +718,7 @@ REPOSITORY_NAME = 'WordPress-Android'
       download_path: download_path
     )
 
+    # TODO: Remove this call after a few sprints of validating that that the `skip_upload_metadata: options[:track] != 'production'` recently added to `upload_build_to_play_store` lane works well
     android_create_xml_release_notes(download_path: download_path, build_number: "#{options[:build_number]}", locales: RELEASE_NOTES_LOCALES)
     sh("git add #{download_path} && git commit -m \"Update WordPress metadata translations for #{options[:version]}\" && git push origin HEAD")
```

Comment:
Hoping to remove that very soon now that I've enabled `skip_upload_changelogs: false` on line 580 above which makes the CI update the release notes in PlayStore automatically for us, and will avoid the need to copy/paste that generated XML file in the future.

I decided to keep that around for a sprint or two just to still have the generated `xml` file around as a fallback until I've validated for a release or two that the new automated way works seemlessly, but I definitively plan to remove those calls very soon 🎉 

(PR on the release scenario to remove that step from the checklist incoming soon, btw)